packages/address/src/utilities.ts

```diff
@@ -69,3 +75,24 @@ function getZone(zones: Zone[], zoneCode: string): Zone {
     }
   );
 }
+
+export function formatAddress(address: Address, country: Country): string[] {
```

Comment:
`utilities` feels like a file that should just be for internal usage rather than exposed publicly. What do you reckon to keeping these functions in AddressFormatter.tsx, and exposing them from there - that way AddressFormatter.tsx is the public api and utilities.ts is just for private stuff.

Reply:
All fair comments, I've got no strong opinion about this and I'm not going to force it. I'm happy for this to go out as-is. Thanks for the viewpoint.

Reply:
Thanks for the thorough review, your comments are valuable as they bring other view points on the changes. 😃 

I think I'll split with a new `format.ts` file, and also split the tests as well, to keep it clear and scoped.